params.yaml

```diff
@@ -1,20 +1,28 @@
 # This file contains parameters that are tracked by DVC.
 # 
 # Notes:
-# - YAML anchor (`&<alias>`) and aliases (`*<alias>`) can be used to define an "anchor"-value that can aliased (referenced) somewhere else in the yaml-file, thereby avoiding the need to repeat the definition of a variable in multiple places, .e.g.
-#     and ommit redundancy
+# - YAML anchor (`&<alias>`) and aliases (`*<alias>`) can be used to define an
+#     "anchor"-value that can aliased (referenced) somewhere else in the
+#     yaml-file, thereby avoid redundancy
 # - YAML overwrites (`<<: *<alias>`) are used to insert default parameters into specific
 #     parameter sections and overwrite specific keys there
 # - Parameters set in this file are unpacked in `dvc.yaml` with e.g. ${baseline-defaults}
 #     resulting in each key being prepended with `--` making it possible to pass them as
 #     command line arguments to neural-lam
 #     (https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#dictionary-unpacking)
+#     
 
 # the name of the "experiment" on mlflow that your run(s) will be grouped under
 # change this to something that describes what you'll be working on (for
 # example what the parameter study is about)
 logger_project: A-baseline
 
+# change this to the path of the virtual environment where you have installed
+# neural-lam and dependencies (creating the venv might become part of the dvc
+# pipeline in future, but for now you have to do this manually)
+venv_path: /dcai/users/denlef/git-repos/mllam/mllam-exps/.venv
```

Comment:
@leifdenby  Would it be better to have this user specific variable set in e.g. `.bashrc`? And document that?

Reply:
yes, good point. Let's just keep the "what versions of software are you using" completely out of the baseline definition for now.

I will change so that the venv path must be defined (enforced by a new stage I will add) with a environment variable called `MLLAM_VENV_PATH`